frontend/src/firebase/listeners.ts

```diff
@@ -240,6 +250,17 @@ export default (onFirstFetched: () => void): (() => void) => {
     reportFirstFetchedIfAllFetched();
   });
 
+  const unmountGroupsListener = listenGroupChange(ownerEmail, (snapshot) => {
+    const data = snapshot.empty ? undefined : snapshot.docs;
```

Comment:
Is snapshot is empty, would `snapshot.docs` be an empty array. Now sure why do you special case here?